proto/conjunction.proto

```diff
@@ -0,0 +1,189 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+syntax = "proto3";
+
+import "proto/answer.proto";
+import "proto/concept.proto";
+
+package typedb.protocol;
+
+message ConjunctionStructure {
```

Comment:
Since this is only used for analyze and for structure - i think a better name for the file is probably `structure.proto`?